interchange/DeviceResources.capnp

```diff
@@ -81,23 +81,24 @@ using TileTypeSiteTypeIdx = UInt32;
 
 struct Device {
 
-  name            @0 : Text;
-  strList         @1 : List(Text) $hashSet();
-  siteTypeList    @2 : List(SiteType);
-  tileTypeList    @3 : List(TileType);
-  tileList        @4 : List(Tile);
-  wires           @5 : List(Wire);
-  nodes           @6 : List(Node);
-  primLibs        @7 : Dir.Netlist; # Netlist libraries of Unisim primitives and macros
-  exceptionMap    @8 : List(PrimToMacroExpansion); # Prims to macros expand w/same name, except these
-  cellBelMap      @9 : List(CellBelMapping);
-  cellInversions @10 : List(CellInversion);
-  packages       @11 : List(Package);
-  constants      @12 : Constants;
-  constraints    @13 : Constraints;
-  lutDefinitions @14 : LutDefinitions;
-  parameterDefs  @15 : ParameterDefinitions;
-  wireTypes      @16 : List(WireType);
+  name                  @0 : Text;
```

Comment:
Why is the indenting here changing? Can you submit the reformatting as a separate pull request?

Reply:
Names, IDs, values are aligned - by adding the new field `belChainsDefinitions` which has longer name than any other existing field, I assumed that fields may need to be re-aligned to the new one.